src/core/client/stream/tabs/Profile/CommentHistory/CommentHistory.tsx

```diff
@@ -2,7 +2,15 @@ import { Localized } from "fluent-react/compat";
 import React, { FunctionComponent } from "react";
 
 import { PropTypesOf } from "coral-framework/types";
-import { Button, HorizontalGutter, Typography } from "coral-ui/components";
+import {
+  Button,
+  Flex,
+  HorizontalGutter,
+  Icon,
+  Typography,
+} from "coral-ui/components";
+
+import styles from "./CommentHistory.css";
```

Comment:
Kiwi informed me you can run into styling issues if your local component styles import isn't the final import.

Reply:
good to know! wonder if there's a TS rule we can add.

Reply:
Apparently it's not available for our linting yet, but I have hopes we can automate this at some point. 😄